packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -331,13 +345,23 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
             >
               <StartCallButton
                 className={mobileWithPreview ? startCallButtonStyleMobile : undefined}
-                onClick={startCallHandler}
+                onClick={startCall}
                 disabled={disableStartCallButton}
               />
             </Stack>
           </Stack>
         </Stack>
-        <SidePane mobileView={props.mobileView} updateSidePaneRenderer={props.updateSidePaneRenderer} />
+        <Panel
```

Comment:
This panel also seems to overlap the whole window, do we have a way to make sure it only overlaps on the composite screen and not all of Contoso's application? I believe I brought this up as a possible bug @prabhjot-msft?

Reply:
There's a layer host property set to scope the panel to the composite - lmk if this doesn't solve what you're referring to